include/ocpp/v201/smart_charging.hpp

```diff
@@ -26,6 +26,7 @@ enum class ProfileValidationResultEnum {
     Valid,
     EvseDoesNotExist,
     ExistingChargingStationExternalConstraints,
+    InvalidSecurityProfile,
```

Comment:
Since this is unrelated to smart charging we should not add this to a result enum of smart charging

Reply:
using hardcoded string now